### PR TITLE
Package to serve as a realsense reboot tester

### PR DESCRIPTION
**Pull request by abhijitmajumdar**
A package is added to enable testing of the realsense cameras to make sure the they do not crash after rebooting the system from scratch and launching the cameras with `initial_reset` enabled

### CODE REVIEW

**Comment by 130s** [comment]
Sorry or the very late response. The entire addition in this PR is a new package `realsense_test`, so I don't foresee adding this package will cause side effect on realsense2 package.

CI failed but CI config looks outdated and it's from the upstream, so I won't bother ourselves fixing it for now.

File: realsense_test/Readme.md [hunk 1]
```diff
@@ -0,0 +1,70 @@
...
+#### Reboot w/o sudo for autoboot user
+```
+sudo su
+echo "autoboot ALL=NOPASSWD: /sbin/reboot">>etc/sudoers
+```
+#### Create workspace
+```
+mkdir -p ~/realsense_ws/src
+cd ~/realsense_ws/src
+git clone -b am_realsense_test https://github.com/abhijitmajumdar/realsense.git
```
Comment (130s) [inline realsense_test/Readme.md:21]:
```suggestion
git clone https://github.com/plusone-robotics/realsense.git
```

File: realsense_test/package.xml [hunk 2]
```diff
@@ -0,0 +1,68 @@
+<?xml version="1.0"?>
+<package format="2">
+  <name>realsense_test</name>
+  <version>0.0.0</version>
+  <description>The realsense_test package</description>
+
+  <!-- One maintainer tag required, multiple allowed, one person per tag -->
+  <!-- Example:  -->
+  <!-- <maintainer email="jane.doe@example.com">Jane Doe</maintainer> -->
+  <maintainer email="ros@todo.todo">ros</maintainer>
```
Comment (130s) [inline realsense_test/package.xml:10]:
```suggestion
  <maintainer email="abhijit.majumdar@plusonerobotics.com">Abhijit Majumdar</maintainer>
```

File: realsense_test/package.xml [hunk 3]
```diff
@@ -0,0 +1,68 @@
...
+
+
+  <!-- One license tag required, multiple allowed, one license per tag -->
+  <!-- Commonly used license strings: -->
+  <!--   BSD, MIT, Boost Software License, GPLv2, GPLv3, LGPLv2.1, LGPLv3 -->
+  <license>TODO</license>
```
Comment (130s) [inline realsense_test/package.xml:16]:
```suggestion
  <license>Apache License 2.0</license>
```

**Review by 130s — CHANGES_REQUESTED** [review]
@abhijitmajumdar I like the approach to make the "reboot test" self-contained within the driver package. Let's discuss if we can replace our reboot test with this (with possible need for some modifications that emerged over time).

Just added cosmetic change suggestions.

**Comment by aaronplusone** [comment]
Closing since this is a couple years old and has no recent updates.

**Pull request closed without merging**